package.json

```diff
@@ -40,7 +40,7 @@
       "4.0.0": {
         "cordova-android": ">=10.0.0"
       },
-      "5.0.0": {
+      "6.0.0": {
```

Comment:
This should include a block for 5.0.0 with the supported platforms of the current plugin version:

```json
"4.0.0": {
  "cordova-android": ">=10.0.0"
},
"5.0.0": {
  "cordova-android": ">=10.0.0"
},
"6.0.0": {
  "cordova": ">100"
}
```

Reply:
Added here: https://github.com/apache/cordova-plugin-media-capture/pull/272/commits/3ed0a55e555c4c08e1bf97cdfa007767cb46ed0b